source/main/environment.js

```diff
@@ -82,7 +82,7 @@ export const environment: Environment = Object.assign(
     isWindows,
     isMacOS,
     isLinux,
-    stateDirectoryPath,
+    stateDirectoryPath: '',
```

Comment:
@DeeJayElly I have on purpose used empty string for the value of state directory. It should be fixed in the scope of this task: https://iohk.myjetbrains.com/youtrack/issue/DDW-676

Reply:
The `stateDirectoryPath` property should be removed from the `environment.js`.